### ISSUE TITLE
Long integers change to scientific notation in XML

### ISSUE THREAD

**Issue by debugmaster** [smart-fun/XmlToJson#6]
Hey. First, great work and idea for this library. :)

I found an issue while converting (JSON->XML) long integers directly from the `JSONObject`. This is a simple scenario:

```
String input =
    "<?xml version='1.0' encoding='UTF-8' standalone='yes' ?>" +
    "<map>" +
        "<long name=\"timestamp\" value=\"1498094219318\" />" +
    "</map>";
XmlToJson json = new XmlToJson.Builder(input).build();
JsonToXml xml = new JsonToXml.Builder(json.toJson())
        .forceAttribute("/map/long/name")
        .forceAttribute("/map/long/value")
        .build();
String output = xml.toString();
System.out.println(input + "\n" + output);
```

The result is:
```xml
<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><map><long name="timestamp" value="1498094219318" /></map>
<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><map><long name="timestamp" value="1.498094219318E12" /></map>
```

It does work fine if I do `json.toJson().toString()` instead of `json.toJson()`.

**Comment by smart-fun** [comment]
Hey Camilo!

thanks for using XmlToJson and thanks a lot for reporting this issue!

I investigated and this is how the JSON library (org.json, not mine) handles numbers. So I did a workaround in my library to detect Long numbers hidden in a Double ;)

please give it a try and tell me if all is ok on your side.
**compile 'com.github.smart-fun:XmlToJson:1.3.3'**

Thanks again!

Arnaud.

**Comment by debugmaster** [comment]
I thank you :)

**Comment by smart-fun** [comment]
Thanks a lot for your help. All has been merged and released in version 1.3.5.

Arnaud.
